src/components/Menu.jsx

```diff
@@ -54,38 +56,51 @@ const MenuPage = () => {
             <Menu.Item
               name="Starters"
               data-cy="starter-tab"
-              active={activeCategory === "starters"}
-              onClick={() => setActiveCategory("starters")}
+              active={activeCategory === 'starters'}
+              onClick={() => setActiveCategory('starters')}
             />
             <Menu.Item
               name="Main Courses"
               data-cy="main-courses-tab"
-              active={activeCategory === "main-menu"}
-              onClick={() => setActiveCategory("main_courses")}
+              active={activeCategory === 'main-menu'}
+              onClick={() => setActiveCategory('main_courses')}
             />
             <Menu.Item
               name="Desserts"
               data-cy="desserts-tab"
-              active={activeCategory === "desserts"}
-              onClick={() => setActiveCategory("desserts")}
+              active={activeCategory === 'desserts'}
+              onClick={() => setActiveCategory('desserts')}
             />
             <Menu.Item
               name="Sides"
               data-cy="sides-tab"
-              active={activeCategory === "sides"}
-              onClick={() => setActiveCategory("sides")}
+              active={activeCategory === 'sides'}
+              onClick={() => setActiveCategory('sides')}
             />
             <Menu.Item
               name="Drinks"
               data-cy="drinks-tab"
-              active={activeCategory === "drinks"}
-              onClick={() => setActiveCategory("drinks")}
+              active={activeCategory === 'drinks'}
+              onClick={() => setActiveCategory('drinks')}
```

Comment:
all this " changing to ' is not good for PR reviewers. Do everyone in a team have the same configuration of Prettier VSCode plugin?

Reply:
happens, there is a way to create a config file for the project, so every one who work on the project has the same settings. Its in the workspace settings.

Reply:
Awesome